server/channels/app/busy_test.go

```diff
@@ -123,7 +123,9 @@ type ClusterMock struct {
 
 func (c *ClusterMock) SendClusterMessage(msg *model.ClusterMessage) {
 	var sbs model.ServerBusyState
-	json.Unmarshal(msg.Data, &sbs)
+	if err := json.Unmarshal(msg.Data, &sbs); err != nil {
+		return
+	}
```

Comment:
Simply returning could obfuscate why a test is failing, The test should either `panic` or we add a `*testing.T` to `ClusterMock` and simply check using `require.NoError`. I prefer the later.

Reply:
Done!

Reply:
```go
func TestSendClusterMessage(t *testing.T) {
    busy := &Busy{}
    clusterMock := &ClusterMock{
        Busy: busy,
        t:    t,
    }

    msg := &model.ClusterMessage{
        Data: []byte(`{"some": "data"}`),
    }

    clusterMock.SendClusterMessage(msg)
}
```

is this good?